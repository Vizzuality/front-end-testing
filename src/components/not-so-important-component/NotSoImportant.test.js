import React from 'react';
import renderer from 'react-test-renderer';

import Component from './';

describe('NotSoImportant component', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(<Component />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
