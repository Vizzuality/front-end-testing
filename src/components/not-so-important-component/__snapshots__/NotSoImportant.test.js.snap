// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotSoImportant component matches snapshot 1`] = `
<div
  className="im-not-important"
>
  shruuugssss
</div>
`;
